.github/workflows/minikube-k8s-test.yml

```diff
@@ -70,3 +70,34 @@ jobs:
           fi
           echo "logs from pod to make sure:"
           cat pod.log
+
+  cypress-tests:
+    name: Run Cypress Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 21
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y xvfb libnss3
+          sudo apt install wget
+          wget https://download.oracle.com/java/21/latest/jdk-21_linux-x64_bin.tar.gz
+          tar -xvf jdk-21_linux-x64_bin.tar.gz
+          cd jdk-21.0.2
+          sudo mkdir -p /usr/local/jdk-21.0.2
+          sudo mv * /usr/local/jdk-21.0.2
+          export JAVA_HOME=/usr/local/jdk-21.0.2
+          export PATH=$JAVA_HOME/bin:$PATH
+          source ~/.bashrc
+          echo 'export JAVA_HOME=/usr/local/jdk-21.0.2' >> ~/.bashrc
+          echo 'export PATH=$JAVA_HOME/bin:$PATH' >> ~/.bashrc
+          source ~/.bashrc
+          npm install
+          sudo java --version
+          ./mvn spring-boot:run
```

Comment:
see the unchanged code above: we already have everything running on minikube. CAn you launch the cypress tests against what is launched above please?

Reply:
I'm sorry I didn't had a look on it, I'll work on it